src/resources/MachineLearning/MachineLearning.ts

```diff
@@ -45,6 +47,7 @@ export default class MachineLearning extends Resource {
         this.relevanceGenerativeAnsweringConfig = new RelevanceGenerativeAnsweringConfiguration(api, serverlessApi);
         this.semanticEncoderConfig = new SemanticEncoderConfiguration(api, serverlessApi);
         this.modelDetailedInfo = new ModelDetailedInfo(api, serverlessApi);
+        this.modelAssociations = new ModelAssociations(api, serverlessApi);
```

Comment:
Missing a test for this property on the resource.

https://github.com/coveo/platform-client/blob/master/src/resources/MachineLearning/tests/MachineLearning.spec.ts

Reply:
fixed